### DISCUSSION TITLE
Delta by Etoro

### DISCUSSION THREAD

**Discussion by navalboyblue** [community/community#135826] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a TCR application submitted but I went ahead and upgraded the legacy connections in the application Delta by Etoro and by doing so the information goes two ways. Submitting a manual trade now actually process like any market or limit order. They need a disclaimer now and the clock cycle timer needs to be async with the clock cycle and the connections. Where do i submit my invoice for 160,379? Both as time in and as a refund. Two inovinces.

**Comment by ebndev** [comment]
Hi @navalboyblue, welcome to the GitHub Community!

It looks like you're looking for support from [eToro](https://help.etoro.com/s/?language=en_GB). You might want to try reaching out via their support page linked.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
